src/routes/filter.ts

```diff
@@ -1,11 +1,23 @@
 import * as Router from 'koa-router';
 
-import { getBodyTypes, getMarks, getModels } from '../controllers/filterController';
+import {
+  getBodyTypes,
+  getMarks,
+  getModels,
+  getSavedFilters,
+  removeAllSavedFilters,
+  removeSavedFilterById,
+  saveFilter
+} from '../controllers/filterController';
 
 const router = new Router();
 
 router.get('/marks', getMarks);
 router.get('/bodyTypes', getBodyTypes);
 router.post('/models', getModels);
+router.get('/saved', getSavedFilters);
+router.post('/saved', saveFilter);
+router.delete('/saved/all', removeAllSavedFilters);
```

Comment:
What's the difference between /all and simple getSavedFilters? Usually the GET /entity endpoint returns the whole list by default.